azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/HandlerFactoryImpl.java

```diff
@@ -14,13 +14,11 @@
 import org.apache.maven.plugin.MojoExecutionException;
 
 public class HandlerFactoryImpl extends HandlerFactory {
-    public static final String RUNTIME_CONFIG_CONFLICT = "Conflict settings found. <javaVersion>, <linuxRuntime>" +
-            "and <containerSettings> should not be set at the same time.";
-    public static final String NO_RUNTIME_HANDLER = "Not able to process the runtime stack configuration; " +
-            "please check <javaVersion>, <linuxRuntime> or <containerSettings> tag in pom.xml";
+    public static final String RUNTIME_CONFIG_CONFLICT = "Conflict settings found. <javaVersion>, <linuxRuntime>"

```

Comment:
do we have to throw the conflict exception, or, just pick/check the right configuration according to the deployment type?

Reply:
runtime is for the Linux, Windows or Containers. It's hard to infer the runtime according to the deployment type user is using.

Reply:
@jdneo I think we should simplify this. Instead of <linuxRuntime>, we should have <environment>, and then choices being: windows, linux, container.

Reply:
@brunoborges Agree, Let’s put this in #200. We can focus on the configuration in the next release after jar deploy.